Functions.Templates/Templates/HttpPOST(CRUD)Queue-FSharp/metadata.json

```diff
@@ -1,12 +1,13 @@
 {
   "defaultFunctionName": "HttpPOST-CRUD-Queue-FSharp",
-  "description": "$HttpPOST(CRUD)Queue-FSharp_description",
-  "name": "HttpPOST(CRUD)Queue",
+  "description": "$HttpPOST(CRUD)Queue_description",
+  "name": "HTTP Post with Queue",
```

Comment:
Should be 'queue' or 'Azure Storage Queue'?

Reply:
@mattchenderson

Reply:
Either of Chris' suggestions works. My preference is "Http POST with Azure Storage queue"